pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/AdminResource.java

```diff
@@ -654,6 +644,16 @@ private void internalCreatePartitionedTopicToReplicatedClustersInBackground(int
                 });
     }
 
+    protected CompletableFuture<Void> internalCreatePartitionedTopicForRemoteCluster(String cluster,
+                                                                                     int numPartitions) {
+        return pulsar().getPulsarResources().getClusterResources()
+                .getClusterAsync(cluster)
+                .thenCompose(clusterDataOp -> ((TopicsImpl) pulsar().getBrokerService()
+                        .getClusterPulsarAdmin(cluster, clusterDataOp).topics())
+                        .createPartitionedTopicAsync(topicName.getPartitionedTopicName(), numPartitions, true, null));
```

Comment:
We should care about two scenarios:
- the partitioned topic on the remote cluster already exists.
- the partition count of the existing topic on the remote cluster is different from the current cluster.

BTW, we can handle these scenarios in a separate PR.